modules/dr-restoring-cluster-state.adoc

```diff
@@ -232,6 +232,194 @@ If the status is `Pending`, or the output lists more than one running etcd pod,
+    instanceState: stopped
+    kind: AWSMachineProviderStatus
+----
+
+... Change the `metadata.name` field to a new name.
++
+It is recommended to keep the same base name as the old machine and change the ending number to the next available number. In this example, clustername-8qw5l-master-0 is changed to clustername-8qw5l-master-3.
++
+For example:
++
+[source,terminal]
+----
+apiVersion: machine.openshift.io/v1beta1
+kind: Machine
+metadata:
+  ...
+  name: clustername-8qw5l-master-3
+  ...
+----
+
+... Update the `metadata.selfLink` field to use the new machine name from the previous step.
++
+[source,terminal]
+----
+apiVersion: machine.openshift.io/v1beta1
+kind: Machine
+metadata:
+  ...
+  selfLink: /apis/machine.openshift.io/v1beta1/namespaces/openshift-machine-api/machines/clustername-8qw5l-master-3
+  ...
+----
+
+... Remove the `spec.providerID` field:
++
+[source,terminal]
+----
+providerID: aws:///us-east-1a/i-0fdb85790d76d0c3f
+----
+
+... Remove the `metadata.annotations` and `metadata.generation` fields:
++
+[source,terminal]
+----
+annotations:
+  machine.openshift.io/instance-state: running
+...
+generation: 2
+----
+
+... Remove the `metadata.resourceVersion` and `metadata.uid` fields:
++
+[source,terminal]
+----
+resourceVersion: "13291"
+uid: a282eb70-40a2-4e89-8009-d05dd420d31a
+----
+
+.. Delete the machine of the lost control plane host:
++
+[source,terminal]
+----
+oc delete machine -n openshift-machine-api clustername-8qw5l-master-0 <1>
+----
+<1> Specify the name of the control plane machine for the lost control plane host.
+
+.. Verify that the machine was deleted:
++
+[source,terminal]
+----
+$ oc get machines -n openshift-machine-api -o wide
+----
++
+Example output:
++
+[source,terminal]
+----
+NAME                                        PHASE     TYPE        REGION      ZONE         AGE     NODE                           PROVIDERID                              STATE
+clustername-8qw5l-master-1                  Running   m4.xlarge   us-east-1   us-east-1b   3h37m   ip-10-0-143-125.ec2.internal   aws:///us-east-1b/i-096c349b700a19631   running
+clustername-8qw5l-master-2                  Running   m4.xlarge   us-east-1   us-east-1c   3h37m   ip-10-0-154-194.ec2.internal   aws:///us-east-1c/i-02626f1dba9ed5bba  running
+clustername-8qw5l-worker-us-east-1a-wbtgd   Running   m4.large    us-east-1   us-east-1a   3h28m   ip-10-0-129-226.ec2.internal   aws:///us-east-1a/i-010ef6279b4662ced   running
+clustername-8qw5l-worker-us-east-1b-lrdxb   Running   m4.large    us-east-1   us-east-1b   3h28m   ip-10-0-144-248.ec2.internal   aws:///us-east-1b/i-0cb45ac45a166173b   running
+clustername-8qw5l-worker-us-east-1c-pkg26   Running   m4.large    us-east-1   us-east-1c   3h28m   ip-10-0-170-181.ec2.internal   aws:///us-east-1c/i-06861c00007751b0a   running
+----
+
+.. Create the new machine using the `new-master-machine.yaml` file:
++
+[source,yaml]
+----
+$ oc apply -f new-master-machine.yaml
```

Comment:
used "oc create -f new-mater-machine.yaml" would be better than "oc apply xxx", because "oc apply" will leave some unnecessary msg in new machine, anyway, both are ok regarding to feature itself.

Reply:
@geliu2016  I updated this command, per your input. @hasbro17 tagging you to let you know about this latest update.

Reply:
/lgtm

Reply:
Thanks @geliu2016 !

Reply:
@hexfusion I made a small change from QE after the first SME review...FYI.

Reply:
I believe create will error if the resource exists so if you are using a previous template and creating a new machine with the same name will it fail? @JoelSpeed thoughts?

Reply:
I guess the assumption is that the old resource is deleted beforehand so perhaps this is not a real concern..

Reply:
Sam's correct here, it's safer to use `create` if we are actually creating otherwise we could just be updating a previously created machine, which I don't think is what we want here. I see it's updated now anyway

Reply:
@JoelSpeed Thanks for your input and also @hexfusion! This change has not gone through doc review and has not been merged, so I can change it.
@geliu2016 I will change this command back to the original since you mentioned that either form is correct. Let me know of any feedback. 

Thanks to all!

Reply:
@tmalove I think we agree with Ge here on create using create in this context.

Reply:
This should not be merged as `oc apply`, this should be `oc create` as it is safer. While both work, we should recommend the safer approach within our docs. Please update it to `oc create` before this merges.